_data/sites.json

```
@@ -2736,6 +2736,16 @@
         ]
     },
 
+    {
+        "name": "Chick-fil-A",
+        "url": "https://privacyportal.onetrust.com/webform/63dc78c7-5612-4181-beae-47dead0569ee/01252c81-82df-4db5-9cb8-bf23a62a9a55",
+        "difficulty": "hard",
```

The difficulty should be "medium" because you just have to fill out a form.
```suggestion
        "difficulty": "medium",
```